software/src/services/open_rocket_simulations.py

```diff
@@ -0,0 +1,15 @@
+import os
+JAR_FILE = "/openrocket.jar"
+os.environ['CLASSPATH'] = os.getcwd() + JAR_FILE
+from jnius import autoclass
```

Comment:
Yeah I think that should work, I ran into some weirdness when trying to get this working and thought perhaps the class path needed to be set before the import of `autoclass`, but I think that wasn't the issue